core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/ClientOverrideConfiguration.java

```diff
@@ -141,6 +145,17 @@ public List<ExecutionInterceptor> executionInterceptors() {
         return executionInterceptors;
     }
 
+    /**
+     * The optional scheduled executor service that should be used for scheduling tasks such as async retry attempts
+     * and timeout task.
+     * <p>
+     * <b>The SDK will not automatically close the executor when the client is closed. It is the responsibility of the
```

Comment:
Can we add this note to the builder setter as well since that's where most users will check?

Reply:
done